metaverse_modules/firedrop/index.js

```diff
@@ -33,6 +33,8 @@ export default e => {
 
   let particleSystem = null;
   const particles = [];
+
+  let abort = false;

```

Comment:
Rather than `abort`, we should call this `live`, which is the convention in the rest of the codebase.

The more modern way is to use https://developer.mozilla.org/en-US/docs/Web/API/AbortController

Reply:
Alright :), will use 'live' to keep convention, but can change it to AbortController if required.